html/semantics/embedded-content/media-elements/playing-the-media-resource/playbackRate.html

```diff
@@ -9,11 +9,46 @@
   assert_equals(v.playbackRate, 1);
 }, 'playbackRate initial value');
 
-async_test(function(t) {
+function testPlaybackRateHelper(t, newPlaybackRate) {
   var v = document.createElement('video');
-  v.playbackRate = 2;
-  v.addEventListener('ratechange', t.step_func(function() {
+  var initialRate = v.playbackRate;
+
+  v.addEventListener('ratechange', function() {
```

Comment:
Opened another [pull request](https://github.com/w3c/web-platform-tests/pull/6593) to address this feedback.